src/test/kotlin/mu/internal/KLoggerNameResolverTest.kt

```diff
@@ -21,7 +21,7 @@ class KLoggerNameResolverTest {
         assertEquals("mu.internal.BaseClass\$InnerClass\$Obj", KLoggerNameResolver.name(BaseClass.InnerClass.Obj.javaClass))
         assertEquals("mu.internal.BaseClass\$InnerClass", KLoggerNameResolver.name(BaseClass.InnerClass.CmpObj::class.java))
         assertEquals("mu.internal.BaseClass\$InnerClass", KLoggerNameResolver.name(BaseClass.InnerClass.CmpObj::class.java))
-        assertEquals("mu.internal.Foo\$Bar", KLoggerNameResolver.name(Foo.Bar::class.java))
```

Comment:
Prior to kotlin 1.1 this Foo.Bar was an object reference. But in kotlin 1.1 it's a variable reference (because you have an object and a variable named the same). I think it's either the test semantics or the variable name must be changed.